src/styles/_loading.scss

```
@@ -0,0 +1,48 @@
+// Loading effects
+.loading--container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+}
+
+$offset: 187;
+$duration: 1.4s;
```

Constants are usually declared with uppercase:
```suggestion
$OFFSET: 187;
$DURATION: 1.4s;
```